data-protocols/dsp/dsp-negotiation/dsp-negotiation-transform/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2023 Fraunhofer Institute for Software and Systems Engineering
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Fraunhofer Institute for Software and Systems Engineering - initial API and implementation
+ *
+ */
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api(project(":data-protocols:dsp:dsp-http-spi"))
+    api(project(":data-protocols:dsp:dsp-negotiation:dsp-negotiation-spi"))
+    api(project(":data-protocols:dsp:dsp-transform"))
```

Comment:
this dependency is suspect, I think we'd need a `dsp-transform-spi` module, maybe something for subsequent PRs

Reply:
Noted for a later PR.

Reply:
dependency is not needed. Removed it. Still, this is a valid suggestion, created #2805.